src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,169 @@
+package duke;
+
+import duke.command.Command;
+import duke.command.HelpCommand;
+import duke.command.ListCommand;
+import duke.command.QuitCommand;
+import duke.command.DoneCommand;
+import duke.command.DeleteCommand;
+import duke.command.TodoCommand;
+import duke.command.DeadlineCommand;
+import duke.command.EventCommand;
+import duke.command.FindCommand;
+import duke.exception.EmptyDescriptionException;
+import duke.exception.InvalidCommandException;
+import duke.exception.InvalidFormatException;
+
+import java.util.Arrays;

```

Comment:
Perhaps it would be better to move the `import java.util.Arrays;` statement to before the `import duke.command.Command` statement?
I got flagged for this by checkstyle myself.